app/models/calculator/pageviews_per_visit.rb

```diff
@@ -0,0 +1,23 @@
+class Calculator::PageviewsPerVisit
+  attr_reader :pageviews, :unique_pageviews
```

Comment:
Don't think we need these attr_readers?

Reply:
Addressed in https://github.com/alphagov/content-data-admin/pull/457/commits/5ad881de50f86420b787384b7200c1815d78c031